### ISSUE TITLE
Rule suggestion - warn when custom error message is missing in expect function

### ISSUE THREAD

**Issue by MindaugasMateika** [playwright-community/eslint-plugin-playwright#51]
What do you think about the rule which should warn about missing custom error messages? I'm not very familiar with rule creation and what rules are ok/not ok so just putting it as a suggestion now.

Docs: https://playwright.dev/docs/test-assertions#custom-expect-message

```diff
+ Good
await expect(page.locator('text=Name'), 'should be logged in').toBeVisible()
```

```diff
- Bad
await expect(page.locator('text=Name')).toBeVisible()
```

```diff
+ Good
await expect.soft(page.locator('text=Name'), 'should be logged in').toBeVisible()
```

```diff
- Bad
await expect.soft(page.locator('text=Name')).toBeVisible()
```

**Comment by mskelton** [comment]
I'm totally fine with this as a rule, so long as it is not enabled by default.

**Comment by mxschmitt** [comment]
+1 like @mskelton said.
